docs/smallfile.md

```diff
@@ -0,0 +1,168 @@
+# Smallfile Benchmark
+
+[Smallfile](https://github.com/distributed-system-analysis/smallfile) is a python-based distributed POSIX workload generator which can be used to quickly measure performance for a variety of metadata-intensive workloads across an entire cluster.
+
+## Running Smallfile Benchmark using Ripsaw
+Once the operator has been installed following the instructions, one needs to modify the clients parameter(which is currently set to 0), to value greater than 0 in  [cr.yaml](../resources/crds/ripsaw_v1alpha1_smallfile_cr.yaml) to run default "create" the test. Also, in addtion to that, smallfile operator is completely dependent on storageclass and storagesize. Please make sure to double check the parameters in CRD file.
```

Comment:
name the CR so it is obvious to someone who is just reading the documents. 

s/[cr.yaml]/[ripsaw_v1alpha1_smallfile_cr.yaml]

Reply:
fixed